code_samples/paging_predicate_sample/README.md

```diff
@@ -0,0 +1,23 @@
+# Paging Predicate Example with Sorting
+
+In this example you will learn how you can use a paging predicate with sorting. In order to define how sorting will work,
+you need a comparator. In order a member to know this comparator, you need to register the comparator in the member config.
+The following describes how to do it:
+
+You can use the sample project in [main](./main) folder:
+
+1. Run the [Main](./main/src/main/java/Main.java) class. The started member will already have the `Comparator` class registered.
+2. Run the sample code in main.js via `node main.js`.
+
+Alternatively, you can start from scratch:
+
+1. Download hazelcast. Extract it somewhere. https://hazelcast.org/imdg/download/
```

Comment:
```suggestion
1. Download Hazelcast. Extract it somewhere. https://hazelcast.org/imdg/download/
```
Can you do the same for other instances?

Also, the link needs to be updated after .com site migration